tests/test_other.py

```diff
@@ -12226,3 +12226,54 @@ def test_rust_gxx_personality_v0(self):
         }
       }
     ''', assert_returncode=NON_ZERO)
+
+  def test_safari_bigint64array_shim(self):
```

Comment:
Is  "shim" the right word here?  How about just `test_bigint64array_polyfill`  (i'm not sure we need to mention safari here).